README.md

```diff
@@ -152,39 +152,53 @@ This default behavior can be overridden with the `default-image-pull-source` con
 
 ##### Image Scanning:
 - alpmdb
-- rpmdb
-- dpkgdb
-- apkdb
-- portage
 - ruby-gemspec
 - python-package
-- php-composer-installed Cataloger
+- php-composer-installed
 - javascript-package
+- dpkgdb
+- rpm-db
 - java
+- graalvm-native-image
+- apkdb
 - go-module-binary
 - dotnet-deps
+- portage
+- nix-store
+- sbom
+- binary
```

Comment:
It looks like this list isn't alphabetical -- would you mind re-sorting it, too?

Reply:
And maybe while you're at it resort the catalogers alphabetically in: https://github.com/anchore/syft/blob/main/syft/pkg/cataloger/cataloger.go ?

Reply:
🤦‍♂️ 

I sorted it, but apparently I goofed!  Let me fix and re-push.